pkg/universe.dagger.io/x/wangwei27494731@gmail.com/kustomize/test/test.cue

```diff
@@ -0,0 +1,36 @@
+package kustomize
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/x/wangwei27494731@gmail.com/kustomize"
+	"encoding/yaml"
+)
+
+dagger.#Plan & {
+	client: {
+		env: KUBECONFIG: string
+		commands: kubeconfig: {
+			name: "cat"
+			args: ["\(env.KUBECONFIG)"]
+			stdout: dagger.#Secret
+		}
```

Comment:
You should retrieve `yaml` with client API instead of `core.#Source` in the package

Reply:
thanks, but I just found the client can use for filesystem, network, env, commands, platform, didn't see yaml API

Reply:
You can just retrieve it as a simple file